### PR TITLE
Changes reflecting deprecated methods: Issue Ticket 665

### PR DESCRIPTION
**Pull request by citygent**
New pull request, had some issues before with CLA and resetting author.

### CODE REVIEW

**Comment by citygent** [comment]
Cripes, I've emailed licence@ as this is getting a bit silly. 
Definitely have the same details in git config and the CLA signature!

**Comment by kborchers** [comment]
@citygent I don't see a signature with your name nor email or any combination of them in our signature list. If you click the details link next to the error it tells you exactly what name and email it expects to find in the list based on the info used to commit the change.

**Comment by citygent** [comment]
screen shot 2015-09-25 at 19 04 54

Do you mean this? Top and bottom match :)

**Comment by kborchers** [comment]
Right, they match because that is where the checker got that information but that information is not in the signature database. My guess is if you sign the CLA with that information this check will turn green.

**Comment by citygent** [comment]
Not sure why this seems to have fixed itself. Maybe I was expecting the CLA check to update quicker than it does. Apologies!

File: page/ajax/jquery-ajax-methods.md [hunk 1]
```diff
@@ -30,32 +30,32 @@ $.ajax({
...
-	// Code to run regardless of success or failure
-	complete: function( xhr, status ) {
-		alert( "The request is complete!" );
-	}
-});
 ```
 
 **Note:** A note about the `dataType` setting: if the server sends back data that is in a different format than you specify, your code may fail, and the reason will not always be clear, because the HTTP response code will not show an error. When working with Ajax requests, make sure your server is sending back the data type you're asking for, and verify that the `Content-type` header is accurate for the data type. For example, for JSON data, the `Content-type` header should be `application/json`.
 
+**Deprecation:** Deprecation Notice: The jqXHR.done(), jqXHR.fail(), and jqXHR.always() callbacks are used above, but you may see the older jqXHR.success(), jqXHR.error(), and jqXHR.complete() callbacks, and should update the code accordingly: As of jQuery 1.8 these are deprecated and will eventually be removed.
```
Comment (arthurvr) [inline page/ajax/jquery-ajax-methods.md]: This is more something for API docs IMHO. Keep in mind that most people are here to learn and use api.jquery.com as a reference. I feel like we shouldn't bother them with deprecated stuff they should be aware of just yet.

File: page/ajax/jquery-ajax-methods.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ Set to `false` if the request should be sent synchronously. Defaults to `true`.
 
 Whether to use a cached response if available. Defaults to `true` for all `dataType`s except "script" and "jsonp". When set to `false`, the URL will simply have a cachebusting parameter appended to it.
 
-#### complete
+#### always
```
Comment (arthurvr) [inline page/ajax/jquery-ajax-methods.md]: Wouldn't it be make more sense to have `done` first?

**Comment by citygent** [comment]
Apologies it has taken so long to address your notes, @arthurvr

**Comment by AurelioDeRosa** [comment]
LGTM :+1:.

**Pull request closed without merging**